mappings/net/minecraft/inventory/Inventory.mapping

```diff
@@ -1,28 +1,41 @@
 CLASS net/minecraft/class_1263 net/minecraft/inventory/Inventory
-	METHOD method_18861 countInInv (Lnet/minecraft/class_1792;)I
+	METHOD method_18861 count (Lnet/minecraft/class_1792;)I
+		COMMENT Determines the total number of times the specified item occurs in this inventory across all stored stacks.
 		ARG 1 item
-	METHOD method_18862 containsAnyInInv (Ljava/util/Set;)Z
+	METHOD method_18862 containsAny (Ljava/util/Set;)Z
+		COMMENT Determines whether this inventory contains any of the given candidate items.
 		ARG 1 items
 	METHOD method_5431 markDirty ()V
-	METHOD method_5432 onInvClose (Lnet/minecraft/class_1657;)V
+	METHOD method_5432 onClose (Lnet/minecraft/class_1657;)V
 		ARG 1 player
-	METHOD method_5434 takeInvStack (II)Lnet/minecraft/class_1799;
+	METHOD method_5434 removeStack (II)Lnet/minecraft/class_1799;
+		COMMENT Removes a specific number of items from the given slot.
+		COMMENT
+		COMMENT @returns the removed items as a stack
 		ARG 1 slot
 		ARG 2 amount
-	METHOD method_5435 onInvOpen (Lnet/minecraft/class_1657;)V
+	METHOD method_5435 onOpen (Lnet/minecraft/class_1657;)V
 		ARG 1 player
-	METHOD method_5437 isValidInvStack (ILnet/minecraft/class_1799;)Z
+	METHOD method_5437 isValid (ILnet/minecraft/class_1799;)Z
+		COMMENT Returns whether the given stack is a valid input to this inventory at the indicated slot position.
 		ARG 1 slot
 		ARG 2 stack
-	METHOD method_5438 getInvStack (I)Lnet/minecraft/class_1799;
+	METHOD method_5438 getStack (I)Lnet/minecraft/class_1799;
+		COMMENT Fetches the stack currently stored at the given slot. If the slot is empty,
+		COMMENT or is outside the bounds of this inventory, will return {@see ItemStack.EMPTY}.
 		ARG 1 slot
-	METHOD method_5439 getInvSize ()I
-	METHOD method_5441 removeInvStack (I)Lnet/minecraft/class_1799;
+	METHOD method_5439 size ()I
+	METHOD method_5441 removeStack (I)Lnet/minecraft/class_1799;
+		COMMENT Removes the stack currently stored at the indicated slot.
+		COMMENT
+		COMMENT @returns the stack previously stored at the indicated slot.
 		ARG 1 slot
-	METHOD method_5442 isInvEmpty ()Z
-	METHOD method_5443 canPlayerUseInv (Lnet/minecraft/class_1657;)Z
+	METHOD method_5442 isEmpty ()Z
+	METHOD method_5443 isAccessible (Lnet/minecraft/class_1657;)Z
+		COMMENT Determines whether the specified player has access to this inventory. Normally only checks distance and whether this inventory is locked.
 		ARG 1 player
-	METHOD method_5444 getInvMaxStackAmount ()I
-	METHOD method_5447 setInvStack (ILnet/minecraft/class_1799;)V
+	METHOD method_5444 getMaxStackAmount ()I
+	METHOD method_5447 putStack (ILnet/minecraft/class_1799;)V
```

Comment:
```suggestion
	METHOD method_5447 setStack (ILnet/minecraft/class_1799;)V
```

Reply:
`putStack` was intended to be a parallel to `List.put` which updates a value at a specific position.

Reply:
I'll see if I can change it.

Reply:
That method is `List.set`, though: https://docs.oracle.com/en/java/javase/13/docs/api/java.base/java/util/List.html#set(int,E)

There's no `List.put`.